sources/FrozenConfig.move

```diff
@@ -0,0 +1,106 @@
+/// The module provide configuration for frozen configuration.
+module StarcoinFramework::FrozenConfig {
+    use StarcoinFramework::ACL;
+    use StarcoinFramework::Config;
+    use StarcoinFramework::Errors;
+    use StarcoinFramework::Signer;
+    use StarcoinFramework::Timestamp;
+    use StarcoinFramework::CoreAddresses;
+
+    friend StarcoinFramework::FrozenConfigStrategy;
+
+    spec module {
+        pragma verify = false;
+        pragma aborts_if_is_strict = true;
+    }
+
+    struct FrozenConfig has copy, drop, store {
+        frozen_global_txn: bool,
+        frozen_account_list: ACL::ACL
+    }
+
+    const ERR_CONFIG_NOT_EXISTS: u64 = 101;
+
+    public fun initialize(sender: &signer, frozen_account_list: ACL::ACL) {
```

Comment:
We can add `spec` to a function to make sure it can be executed properly. For a config, it can be like: 
`        spec initialize {
            aborts_if Signer::address_of(account) != CoreAddresses::GENESIS_ADDRESS();
            aborts_if exists<Config::Config<FrozenConfig>>(Signer::address_of(account));
            aborts_if exists<Config::ModifyConfigCapabilityHolder<FrozenConfig>>(Signer::address_of(account));
            ensures exists<Config::Config<FrozenConfig>>(Signer::address_of(account));
            ensures
                exists<Config::ModifyConfigCapabilityHolder<FrozenConfig>>(
                    Signer::address_of(account),
                );
        }`